scrapi/consumers/calpoly/README.md

```diff
@@ -0,0 +1,8 @@
+CalPoly
+=======
+
+Consumer for the Digital Commons @ CalPoly API for the SHARE project
```

Comment:
Could we remove all the readmes?
Maybe just move them into the doc string of __init__?